community/detectors/papercut_ng_mf_CVE_2023_27350/src/main/java/com/google/tsunami/plugins/papercut/PapercutNGMFHelper.java

```diff
@@ -0,0 +1,137 @@
+package com.google.tsunami.plugins.papercut;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.flogger.GoogleLogger;
+import com.google.protobuf.ByteString;
+import com.google.tsunami.common.data.NetworkServiceUtils;
+import com.google.tsunami.common.net.http.HttpClient;
+import com.google.tsunami.common.net.http.HttpHeaders;
+import com.google.tsunami.common.net.http.HttpRequest;
+import com.google.tsunami.common.net.http.HttpResponse;
+import com.google.tsunami.proto.NetworkService;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PapercutNGMFHelper {
+
+  private final NetworkService networkService;
+  private final GoogleLogger logger;
+  private final HttpClient httpClient;
+  public String JSESSION_ID = "";
+  private String root_url = "";
+  private String base_app_url = "";
```

Comment:
Minor styling issue: these should all be in camel case, match pattern '^[a-z][a-zA-Z0-9]*_?$'. [MemberName]

Only the static variables are in capitalized format with underscores.